### PR TITLE
Master wowl error mcm

### PR DESCRIPTION
**Pull request by mcm-odoo**

### CODE REVIEW

File: addons/web/static/tests/core/errors/error_service_tests.js [hunk 1]
```diff
@@ -235,5 +236,42 @@ QUnit.test("handle uncaught CORS errors", async (assert) => {
 
     // CORS error event has no colno, no lineno and no filename
     const errorEvent = new ErrorEvent("error", { error });
-    errorCb(errorEvent);
+    await errorCb(errorEvent);
+});
+
+QUnit.test("check retry", async (assert) => {
```
Comment (ged-odoo) [inline addons/web/static/tests/core/errors/error_service_tests.js:242]: yay, thanks for adding the test

**Comment by ged-odoo** [comment]
robodoo r+

**Comment by ged-odoo** [comment]
robodoo retry

**Comment by mcm-odoo** [comment]
robodoo retry

**Pull request closed without merging**